src/components/bar.js

```diff
@@ -199,9 +199,7 @@ class Bar extends Component {
                     icon='folder'
                     text='Create folder'
                   />
-                  <Dropdown.Item>
-                    <Upload />
-                  </Dropdown.Item>
+                  <Upload />
```

Comment:
Upload ke andar bhi ek Dropdown.Item hai vo code bhot ganda lagega if you put Dropdown.Item inside a Dropdown.Item
isko poora shi kr de